etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -379,8 +370,8 @@ f5_vtep_selfip_name = 'vtep'
 #
 # This is a boolean entry which determines if they BIG-IP® will use
 # L2 Population service to update its fdb tunnel entries. This needs
-# to be setup in accordance with the way the other tunnel agents are
-# setup.  If the BIG-IP® agent and other tunnel agents don't match
+# to be set up in accordance with the way the other tunnel agents are
+# set up.  If the BIG-IP® agent and other tunnel agents don't match
 # the tunnel setup will not work properly.
 #
 l2_population = True
```

Comment:
Can we change 'Global Routing Mode' to 'Global Routed Mode' so it matches the actual setting?